src/main/java/no/nav/foreldrepenger/soknad/innsending/fordel/dokument/DokumentTypeId.java

```diff
@@ -13,57 +12,40 @@
-
-    // Vedlegg fra brukerdialog og fyllut-sendinn
-    LEGEERKLÆRING("LEGEERKLÆRING", "I000023", "Legeerklæring"),
-    DOK_INNLEGGELSE("DOK_INNLEGGELSE", "I000037", "Dokumentasjon av innleggelse i helseinstitusjon"),
-    DOK_MORS_UTDANNING_ARBEID_SYKDOM("DOK_MORS_UTDANNING_ARBEID_SYKDOM", "I000038", "Dokumentasjon av mors utdanning arbeid eller sykdom"),
-    DOK_MILITÆR_SIVIL_TJENESTE("DOK_MILITÆR_SIVIL_TJENESTE", "I000039", "Dokumentasjon av militær- eller siviltjeneste"),
-    DOKUMENTASJON_AV_TERMIN_ELLER_FØDSEL("DOKUMENTASJON_AV_TERMIN_ELLER_FØDSEL", "I000041",
-        "Dokumentasjon av termindato (lev. kun av mor), fødsel eller dato for omsorgsovertakelse"),
-    DOKUMENTASJON_AV_OMSORGSOVERTAKELSE("DOKUMENTASJON_AV_OMSORGSOVERTAKELSE", "I000042", "Dokumentasjon av dato for overtakelse av omsorg"),
-    DOK_ETTERLØNN("DOK_ETTERLØNN", "I000044", "Dokumentasjon av etterlønn/sluttvederlag"),
-    BESKRIVELSE_FUNKSJONSNEDSETTELSE("BESKRIVELSE_FUNKSJONSNEDSETTELSE", "I000045", "Beskrivelse av funksjonsnedsettelse"),
-    BEKREFTELSE_FRA_STUDIESTED("BEKREFTELSE_FRA_STUDIESTED", "I000061", "Bekreftelse fra studiested/skole"),
-    BEKREFTELSE_VENTET_FØDSELSDATO("BEKREFTELSE_VENTET_FØDSELSDATO", "I000062", "Bekreftelse på ventet fødselsdato"),
-    FØDSELSATTEST("FØDSELSATTEST", "I000063", "Fødselsattest"),
-    BEKREFTELSE_FRA_ARBEIDSGIVER("BEKREFTELSE_FRA_ARBEIDSGIVER", "I000065", "Bekreftelse fra arbeidsgiver"),
-    DOKUMENTASJON_ALENEOMSORG("DOKUMENTASJON_ALENEOMSORG", "I000110", "Dokumentasjon av aleneomsorg"),
-    BEGRUNNELSE_SØKNAD_ETTERSKUDD("BEGRUNNELSE_SØKNAD_ETTERSKUDD", "I000111", "Dokumentasjon av begrunnelse for hvorfor man søker tilbake i tid"),
-    DOKUMENTASJON_INTRODUKSJONSPROGRAM("DOKUMENTASJON_INTRODUKSJONSPROGRAM", "I000112", "Dokumentasjon av deltakelse i introduksjonsprogrammet"),
-    DOKUMENTASJON_FORSVARSTJENESTE("DOKUMENTASJON_FORSVARSTJENESTE", "I000116",
-        "Bekreftelse på øvelse eller tjeneste i Forsvaret eller Sivilforsvaret"),
-    DOKUMENTASJON_NAVTILTAK("DOKUMENTASJON_NAVTILTAK", "I000117", "Bekreftelse på tiltak i regi av Arbeids- og velferdsetaten"),
+    SØKNAD_SVANGERSKAPSPENGER("I000001", "Søknad om svangerskapspenger"),
+    SØKNAD_FORELDREPENGER_ADOPSJON("I000002", "Søknad om foreldrepenger ved adopsjon"),
+    SØKNAD_ENGANGSSTØNAD_FØDSEL("I000003", "Søknad om engangsstønad ved fødsel"),
+    SØKNAD_ENGANGSSTØNAD_ADOPSJON("I000004", "Søknad om engangsstønad ved adopsjon"),
+    SØKNAD_FORELDREPENGER_FØDSEL("I000005", "Søknad om foreldrepenger ved fødsel"),
+    FLEKSIBELT_UTTAK_FORELDREPENGER("I000006", "Utsettelse eller gradert uttak av foreldrepenger (fleksibelt uttak)"),
+    FORELDREPENGER_ENDRING_SØKNAD("I000050", "Søknad om endring av uttak av foreldrepenger eller overføring av kvote"),
+
+    // Vedlegg
+    INNTEKTSOPPLYSNING_SELVSTENDIG("I000007", "Inntektsopplysninger om selvstendig næringsdrivende og/eller frilansere som skal ha foreldrepenger eller svangerskapspenger"),
```

Comment:
Denne gjengen stammer mest fra scanning-tiden.
Det er 2 hensyn her
* Forenkling - kun de skjema / navn som er støttet av frontend (som har en tittel og et filnavn)
* Mulighet til å kjenne ukjente titler (mest behov for scanning)

Hva trengs her? Fyllut-sendinn-titler er ikke aktuelt her.

Reply:
Jeg har sjekket hva som brukes i selvbetjening frontend og fjernet alt som ikke var der.

Dokumenttypeid (I000XYZ) sendes ned fra frontend så denne trenger vi. Ellers brukes vel tittelen for vedlegg.
Fyllut-sendinn-titler er ikke aktuelle skjønner jeg ikke? Hva mener du?

Reply:
Det er noe kode for å sette dokumenttittel = beskrivelse dersom skjemanummer = I000060.
Usikker på om frontend sender med noen verdi i beskrivelse. 
Tror ikke vi vi prøver gjenkjenne noe fra beskrivelse - der kan man fx finne Inntektsmelding dersom fått fra AG og tatt bilde og vedlagt som ANNET med beskrivelse Inntektsmelding - noe som kan gi funky feil

Reply:
Tar videre diskusjon på slack: https://nav-it.slack.com/archives/C09039RR84B/p1756804065317579